### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by Matthensley777**
Should be finished assignment.

### CODE REVIEW

**Comment by zoeames** [comment]
code works great!  You just need to do a little bit of code cleanup.  Get rid of the blank lines inside functions.  Only have one blank line between functions.  Checking the indenting in your html file

File: index.html [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 </head>
 <body>
-<div>
-    <input type="text" id="size" style="width: 40%" placeholder="How tall should the tree be?">
-</div>
-<div>
-    <input type="text" id="char" style="width: 40%" placeholder="Which character should be used?">
-</div>
+	<div>
+    	<input type="text" id="size" style="width: 40%" placeholder="How tall should the tree be?">
```
Comment (morecallan) [inline index.html]: (these inputs should actually be indented one line because they are a child of the div (because it wraps them)

File: main.js [hunk 2]
```diff
@@ -5,20 +5,12 @@ function christmasTreeVars() {
...
 		var chars = 2*i+1;
 		var spaces = treeObject.height-(i+1);
-		
 		console.log(" ".repeat(spaces)+treeObject.char.repeat(chars))
-		
-	 }
-
-
-
+		}
```
Comment (morecallan) [inline main.js]: This bracket is indented one too far.

**Review by morecallan — COMMENTED** [review]
@Matthensley777 suuuuuper nit-picky indentation comments but you'll thank me later.

**Comment by morecallan** [comment]
👍 all good!

**Pull request merged**